src/material/checkbox/checkbox.scss

```diff
@@ -288,6 +288,10 @@ $_mat-checkbox-mark-stroke-size: 2 / 15 * checkbox-common.$size !default;
     variables.$linear-out-slow-in-timing-function, opacity checkbox-common.$transition-duration
     variables.$linear-out-slow-in-timing-function;
 
+  // force browser to show background-color when using the print function
+  -webkit-print-color-adjust: exact;
+  color-adjust: exact;
+
```

Comment:
I think that we'll need the same fixes for the `mdc-radio` and `mdc-checkbox`.

Reply:
Should I open a new pr/issue in the MDC repository or update the material/experimental mdc components in this pull request?

Reply:
It's fine to fix it on our end for now. If MDC ends up resolving it on their end, we can remove our workaround.

Reply:
Added an additional commit for the mdc-checkbox. The mdc-radio was printed fine because it uses a border for the inner circle.